CONTRIBUTING.md

```diff
@@ -3,6 +3,16 @@ Contributing to Testem
 
 Open source is all about DIY! If you want something fixed, it's sometimes faster to just roll your sleeves up, not to mention lots more rewarding. This doc will give you some pointers on where to look when you poke around Testem's source code.
 
+Getting Started
+---------------
+
+* Fork and checkout [github.com/testem/testem](https://github.com/testem/testem)
+* Run `npm install` and `npm test` to make sure you're off to a good start
+
+> Testem expects `PhantomJS` to be in the PATH, if you don't have one,
+> either install it globally via `sudo npm install -g phantomjs`
```

Comment:
Could you remove the `sudo` here? Reasoning: https://github.com/testem/testem/pull/598#issuecomment-121334629